docs/tutorials/object-management-kubectl/object-management.md

```diff
@@ -71,12 +71,19 @@ object in either YAML or JSON.
 The user provides an operation (create, replace, delete), one or more files,
 and flags to the `kubectl` command.
 
-This technique requires a deep understanding of the Kubernetes
-object definitions.
+This technique requires understanding the Kubernetes
+object definitions to create and update them.
```

Comment:
When using imperative commands, the user doesn't need to understand the object definition or how it is laid out in yaml, they just supply the flags.  (e.g. --replicas instead of understanding where the replicas field would need to be specified in the configuration file)